atd-etl/cr3_download/docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-version: '3'
-
 services:
   cr3_download:
     build: .
     volumes:
       - .:/app
     env_file:
       - ./.env
+    # entrypoint: /bin/bash
```

Comment:
I want to leave in this breadcrumb to aid in recalling how to get a shell easily on this image.